app/controllers/api/v2/hosts_controller.rb

```diff
@@ -252,6 +252,10 @@ def disassociate
 
       def power
         valid_actions = PowerManager::SUPPORTED_ACTIONS
+        unless @host.supports_power?
+          return render_message(_('This host does not support power operations'), :status => :unprocessable_entity)
```

Comment:
shouldn't this use `render_error`?

Reply:
`render_error` will generate a custom page, I still want the response to remain simple. I want hammer and UI to be able to parse the response.

Reply:
`render_error` doesn't have to do anything with UI. The difference is how the json output looks like, but hammer should be able to process both. It feels to me still that this is more of an error state than just a message, although, we are using the `render_message` and `render_error` inconsistently  here, which I guess would need a new API version to unify. If we wanted to unify the way we render this kind of messages, I would go for render_error layout probably. But leaving the decision up to you: just be aware it has nothing to do with UI and hammer should be able to process the error.